spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -31,27 +31,26 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
 
       omniboxController.suggestions = 100;
       expect(omniboxController.hasSuggestions()).toBe(false);
-    });
 
-    it('should return false for empty Arrays and Objects', () => {
-      omniboxController.suggestions = [];
+      omniboxController.suggestions = {test: 'me'};
       expect(omniboxController.hasSuggestions()).toBe(false);
```

Comment:
Should it error instead of being false? I feel like at some point there should be an error.